app/packages/looker/src/elements/common/actions.ts

```diff
@@ -142,6 +142,29 @@ export const previous: Control = {
   },
 };
 
+
+export const hideLabels: Control = {
+  title: "Hide labels",
+  shortcut: "h",
+  eventKeys: "h",
+  detail: "Toggles visibility of all labels",
+  action: (update, dispatchEvent) => {
+    update(({ options: { showLabels } }) => {
+      if (showLabels) {
+        dispatchEvent("options", { showLabels: false });
```

Comment:
So as you may have noticed, there a quite a few options that define a Looker's state.

The word "options` is also overused. Specifically, the "options" event is for AppConfig options which  have configurable defaults that come from the server

* https://github.com/voxel51/fiftyone/blob/499a53e34c58585b9ac0bf0d7d7e6e5b1030aec4/fiftyone/server/query.py#L207
* https://github.com/voxel51/fiftyone/blob/499a53e34c58585b9ac0bf0d7d7e6e5b1030aec4/fiftyone/core/config.py#L270

The default for this should always be true, though.